otx/algorithms/classification/tasks/inference.py

```diff
@@ -302,6 +302,9 @@ def _init_recipe(self):
 
         if train_type == TrainType.INCREMENTAL:
             recipe = os.path.join(recipe_root, "incremental.yaml")
+        if train_type == TrainType.SELFSUPERVISED:
+            recipe = self.template_file_path.replace(os.path.basename(self.template_file_path), "class_selfsl.yaml")
```

Comment:
This looks like a hack. The selfsl recipe should be located in the same folder as the other ones (inside the MPA repo).